docs/source/main_concepts/module.rst

```diff
@@ -50,6 +50,46 @@ in dynamic parameter storage.
   Currently ``etcd`` is supported as dynamic parameter storage. Etcd connection requires
   parameter values under ``parameters.etcd_config`` to be valid.
 
+Configuring frame output
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In case a Savant module is intended to run alongside **image-** or **video-files** sink :ref:`adapters <main_concepts/adapters:Adapters overview>` its ``output_frame`` parameter must be configured correctly in order to get frame data in the output messages.
+
+1. ``parameters.output_frame.codec`` config node determines which codec will be used to encode output frames. Possible values are **h264**, **h265**, **jpeg**, **png** and **raw-rgba** to skip encoding. For example:
+
+  .. code-block:: YAML
+
+    parameters:
+      output_frame:
+        codec: h264
+
+2. ``parameters.output_frame.encoder_params`` config node allows passing additional encoder-specific parameters to the encoder pipeline element. These are:
```

Comment:
The documentation should describe the parameters for each codec with which the user can set the encoding quality, so that the user does not have to run gst-inspect-1.0 to find them.

Reply:
Listed encoder elements properties explicitly.